backends/cloud/savessyncrequest.cpp

```diff
@@ -113,6 +115,13 @@ void SavesSyncRequest::directoryListedCallback(Storage::ListDirectoryResponse re
 				_filesToUpload.push_back(file.name());
 			else
 				_filesToDownload.push_back(file);
+
+			if (_localFilesTimestamps[name] == DefaultSaveFileManager::INVALID_TIMESTAMP)
+				debug(9, "- uploading file %s, because it is has invalid timestamp", name.c_str());
+			else if (_localFilesTimestamps[name] > file.timestamp())
+				debug(9, "- uploading file %s, because it is %d seconds newer than remote\n\tlocal = %d; \tremote = %d", name.c_str(), _localFilesTimestamps[name] - file.timestamp(), _localFilesTimestamps[name], file.timestamp());
+			else
+				debug(9, "- downloading file %s, because it is %d seconds older than remote\n\tlocal = %d; \tremote = %", name.c_str(), file.timestamp() - _localFilesTimestamps[name], _localFilesTimestamps[name], file.timestamp());
```

Comment:
I am guessing there is a `d` missing after the `remote = %` here.

Reply:
Oof.